pkg/provider/provider.go

```diff
@@ -80,6 +80,17 @@ func DeploymentsParse(deploymentFiles []string, deploymentVars map[string]string
 		}
 	}
 
+	for k, v := range deploymentVars {
+		_, err := os.Stat(v)
+		if err == nil && v != "prombench" {
```

Comment:
this looks wrong, why do we need it?

Reply:
Prombench binary exists in the directory and we are checking if some file with given name exists.
It reads the binary and fails.

Reply:
Sorry I didn't understand what you mean.

Reply:
eg. /home/github/ARG_0 file exists and err will be nil. 

We have `CLUSTER_NAME` whose value is prombench. Prombench binary file named "prombench" exists. As err is nil, the condition gets satisfied and `ioutil.ReadFile()` will read the file.

It tries to read the binary file and assign that value to `CLUSTER_NAME`. This results in failure.

Reply:
this is wrong, 
you can use some file type detection like: https://github.com/h2non/filetype

Reply:
maybe also implement this for the auth file which I added recently

Reply:
We are setting the env variable to the path of the file

Reply:
I didn't understand what you mean?

Reply:
ping